learning_journal/models.py

```diff
@@ -25,6 +27,6 @@ class Entry(Base):
     id = Column(Integer, primary_key=True)
     title = Column(String(length=128, convert_unicode=True), unique=True)
     text = Column(Text(convert_unicode=True))
```

Comment:
This PR is not complete.  I see code for your entry model here that isn't in the diff.  It makes it hard to grade your entire first week of work when I can't see it all in the pull request.